### ISSUE TITLE
[Bug] Crash when loading https://doc.e.foundation/devices/

### ISSUE THREAD

**Issue by Massedil** [mozilla-mobile/fenix#19370]
[comment]: # (Please do your best to search for duplicate issues before filing a new issue so we can keep our issue board clean)
[comment]: # (Every issue should have the exact bug and steps to reproduce described in it. Please do not file feedback list tickets as it is difficult to parse them and address their individual points)
[comment]: # (Read https://github.com/mozilla-mobile/fenix#i-want-to-file-an-issue for more information)

## Steps to reproduce

Just go to https://doc.e.foundation/devices/.

### Expected behavior

No crash.

### Actual behavior

Firefox crashes.

### Device information

* Device vendor / model and Android version: Samsung Galaxy S5
* Firefox for Android version: 88.1.1
 
### Logs 2021-05-03

```
05-03 23:36:49.925 I/a.fennec_fdroi(22469): Background young concurrent copying GC freed 72134(3366KB) AllocSpace objects, 30(600KB) LOS objects, 40% free, 6072KB/10122KB, paused 175us total 171.562ms
05-03 23:36:52.052 I/GeckoSession(22469): handleMessage GeckoView:PageStop uri=null
05-03 23:36:52.060 I/SessionStorage/AutoSave(22469): Save: Load finished
05-03 23:36:52.327 I/a.fennec_fdroi(22469): Background concurrent copying GC freed 121120(5760KB) AllocSpace objects, 2(60KB) LOS objects, 56% free, 4750KB/10MB, paused 142us total 112.391ms
05-03 23:36:59.612 I/ActivityManager(760): Process XXXXXX has died: cch+25 CEM 
05-03 23:36:59.616 I/Zygote  (434): Process 22852 exited due to signal 9 (Killed)
05-03 23:36:59.657 I/libprocessgroup(760): Successfully killed process cgroup uid 10278 pid 22852 in 0ms
05-03 23:36:59.703 I/ActivityManager(760): Process XXXXXX has died: cch+15 CEM 
05-03 23:36:59.709 I/libprocessgroup(760): Successfully killed process cgroup uid 10207 pid 22801 in 0ms
05-03 23:36:59.713 I/FenixApplication(22469): onTrimMemory(), level=15, main=true
05-03 23:36:59.716 I/FenixApplication(22544): onTrimMemory(), level=15, main=false
05-03 23:36:59.717 I/ServiceChildProcess(22544): onTrimMemory(15)
05-03 23:36:59.723 I/Zygote  (434): Process 22801 exited due to signal 9 (Killed)
05-03 23:36:59.757 I/TrimMemoryMiddleware(22469): Trim memory (tabs=14, suspending=0)
05-03 23:36:59.841 I/ActivityManager(760): Process XXXXXX has died: cch+15 CEM 
05-03 23:36:59.842 I/libprocessgroup(760): Successfully killed process cgroup uid 10003 pid 22828 in 0ms
05-03 23:36:59.860 I/Zygote  (434): Process 22828 exited due to signal 9 (Killed)
05-03 23:36:59.900 I/ActivityManager(760): Process XXXXXX has died: cch+5 CEM 
05-03 23:36:59.923 I/Zygote  (434): Process 22873 exited due to signal 9 (Killed)
05-03 23:36:59.958 I/libprocessgroup(760): Successfully killed process cgroup uid 10109 pid 22873 in 56ms
05-03 23:36:59.979 I/ActivityManager(760): Process XXXXXX has died: prev LAST
05-03 23:36:59.986 I/libprocessgroup(760): Successfully killed process cgroup uid 10000 pid 23044 in 0ms
05-03 23:37:00.000 I/Zygote  (434): Process 23044 exited due to signal 9 (Killed)
05-03 23:37:00.051 I/WindowManager(760): WIN DEATH: Window{92ed096 u0 InputMethod}
05-03 23:37:00.057 I/Zygote  (434): Process 22320 exited due to signal 9 (Killed)
05-03 23:37:00.063 W/InputDispatcher(760): channel '92ed096 InputMethod (server)' ~ Consumer closed input channel or an error occurred.  events=0x9
05-03 23:37:00.063 E/InputDispatcher(760): channel '92ed096 InputMethod (server)' ~ Channel is unrecoverably broken and will be disposed!
05-03 23:37:00.063 W/InputDispatcher(760): Attempted to unregister already unregistered input channel '92ed096 InputMethod (server)'
05-03 23:37:00.064 I/ActivityManager(760): Process XXXXXX has died: prcp IMPB
05-03 23:37:00.066 I/libprocessgroup(760): Successfully killed process cgroup uid 10287 pid 22320 in 0ms
05-03 23:37:00.142 D/KeyguardClockSwitch(32243): Updating clock: 23:37
05-03 23:37:00.221 W/Looper  (760): Slow dispatch took 187ms main h=com.android.server.AlarmManagerService$AlarmHandler c=com.android.server.-$$Lambda$AlarmManagerService$2$Eo-D98J-N9R2METkD-12gPs320c@c182c00 m=0
05-03 23:37:00.227 W/ActivityManager(760): Scheduling restart of crashed service com.menny.android.anysoftkeyboard/.SoftKeyboard in 1000ms
05-03 23:37:00.288 W/ActivityManager(760): setHasOverlayUi called on unknown pid: 22320
05-03 23:37:00.363 W/InputMethodManagerService(760): Session failed to close due to remote exception
05-03 23:37:00.363 W/InputMethodManagerService(760): android.os.DeadObjectException
05-03 23:37:00.363 W/InputMethodManagerService(760): 	at android.os.BinderProxy.transactNative(Native Method)
05-03 23:37:00.363 W/InputMethodManagerService(760): 	at android.os.BinderProxy.transact(BinderProxy.java:511)
05-03 23:37:00.363 W/InputMethodManagerService(760): 	at com.android.internal.view.IInputMethodSession$Stub$Proxy.finishSession(IInputMethodSession.java:419)
05-03 23:37:00.363 W/InputMethodManagerService(760): 	at com.android.server.inputmethod.InputMethodManagerService.finishSessionLocked(InputMethodManagerService.java:2366)
05-03 23:37:00.363 W/InputMethodManagerService(760): 	at com.android.server.inputmethod.InputMethodManagerService.clearClientSessionLocked(InputMethodManagerService.java:2357)
05-03 23:37:00.363 W/InputMethodManagerService(760): 	at com.android.server.inputmethod.InputMethodManagerService.clearCurMethodLocked(InputMethodManagerService.java:2384)
05-03 23:37:00.363 W/InputMethodManagerService(760): 	at com.android.server.inputmethod.InputMethodManagerService.onServiceDisconnected(InputMethodManagerService.java:2411)
05-03 23:37:00.363 W/InputMethodManagerService(760): 	at android.app.LoadedApk$ServiceDispatcher.doDeath(LoadedApk.java:1972)
05-03 23:37:00.363 W/InputMethodManagerService(760): 	at android.app.LoadedApk$ServiceDispatcher$RunConnection.run(LoadedApk.java:1987)
05-03 23:37:00.363 W/InputMethodManagerService(760): 	at android.os.Handler.handleCallback(Handler.java:883)
05-03 23:37:00.363 W/InputMethodManagerService(760): 	at android.os.Handler.dispatchMessage(Handler.java:100)
05-03 23:37:00.363 W/InputMethodManagerService(760): 	at android.os.Looper.loop(Looper.java:214)
05-03 23:37:00.363 W/InputMethodManagerService(760): 	at com.android.server.SystemServer.run(SystemServer.java:558)
05-03 23:37:00.363 W/InputMethodManagerService(760): 	at com.android.server.SystemServer.main(SystemServer.java:356)
05-03 23:37:00.363 W/InputMethodManagerService(760): 	at java.lang.reflect.Method.invoke(Native Method)
05-03 23:37:00.363 W/InputMethodManagerService(760): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:491)
05-03 23:37:00.363 W/InputMethodManagerService(760): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:918)
05-03 23:37:00.370 W/InputMethodManagerService(760): Session failed to close due to remote exception
05-03 23:37:00.370 W/InputMethodManagerService(760): android.os.DeadObjectException
05-03 23:37:00.370 W/InputMethodManagerService(760): 	at android.os.BinderProxy.transactNative(Native Method)
05-03 23:37:00.370 W/InputMethodManagerService(760): 	at android.os.BinderProxy.transact(BinderProxy.java:511)
05-03 23:37:00.370 W/InputMethodManagerService(760): 	at com.android.internal.view.IInputMethodSession$Stub$Proxy.finishSession(IInputMethodSession.java:419)
05-03 23:37:00.370 W/InputMethodManagerService(760): 	at com.android.server.inputmethod.InputMethodManagerService.finishSessionLocked(InputMethodManagerService.java:2366)
05-03 23:37:00.370 W/InputMethodManagerService(760): 	at com.android.server.inputmethod.InputMethodManagerService.clearClientSessionLocked(InputMethodManagerService.java:2357)
05-03 23:37:00.370 W/InputMethodManagerService(760): 	at com.android.server.inputmethod.InputMethodManagerService.clearCurMethodLocked(InputMethodManagerService.java:2384)
05-03 23:37:00.370 W/InputMethodManagerService(760): 	at com.android.server.inputmethod.InputMethodManagerService.onServiceDisconnected(InputMethodManagerService.java:2411)
05-03 23:37:00.370 W/InputMethodManagerService(760): 	at android.app.LoadedApk$ServiceDispatcher.doDeath(LoadedApk.java:1972)
05-03 23:37:00.370 W/InputMethodManagerService(760): 	at android.app.LoadedApk$ServiceDispatcher$RunConnection.run(LoadedApk.java:1987)
05-03 23:37:00.370 W/InputMethodManagerService(760): 	at android.os.Handler.handleCallback(Handler.java:883)
05-03 23:37:00.370 W/InputMethodManagerService(760): 	at android.os.Handler.dispatchMessage(Handler.java:100)
05-03 23:37:00.370 W/InputMethodManagerService(760): 	at android.os.Looper.loop(Looper.java:214)
05-03 23:37:00.370 W/InputMethodManagerService(760): 	at com.android.server.SystemServer.run(SystemServer.java:558)
05-03 23:37:00.370 W/InputMethodManagerService(760): 	at com.android.server.SystemServer.main(SystemServer.java:356)
05-03 23:37:00.370 W/InputMethodManagerService(760): 	at java.lang.reflect.Method.invoke(Native Method)
05-03 23:37:00.370 W/InputMethodManagerService(760): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:491)
05-03 23:37:00.370 W/InputMethodManagerService(760): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:918)
05-03 23:37:00.370 W/Looper  (760): Slow dispatch took 149ms main h=android.app.ActivityThread$H c=android.app.LoadedApk$ServiceDispatcher$RunConnection@6b43df m=0
05-03 23:37:00.371 W/Looper  (760): Slow delivery took 270ms main h=com.android.server.AlarmManagerService$AlarmHandler c=null m=4
05-03 23:37:00.371 W/Looper  (760): Drained
05-03 23:37:00.701 I/WindowManager(760): WIN DEATH: Window{466c7c9 u0 org.mozilla.fennec_fdroid/org.mozilla.fenix.HomeActivity}
05-03 23:37:00.701 D/ConnectivityService(760): ConnectivityService NetworkRequestInfo binderDied(NetworkRequest [ LISTEN id=667, [ Transports: WIFI Capabilities: NOT_RESTRICTED&TRUSTED&NOT_VPN&FOREGROUND Uid: 10081] ], android.os.BinderProxy@9d4082c)
05-03 23:37:00.702 I/ActivityManager(760): Process XXXXXX has died: fore TOP 
05-03 23:37:00.706 E/InputDispatcher(760): Window handle Window{466c7c9 u0 org.mozilla.fennec_fdroid/org.mozilla.fenix.HomeActivity} has no registered input channel
05-03 23:37:00.736 E/InputDispatcher(760): Window handle Window{466c7c9 u0 org.mozilla.fennec_fdroid/org.mozilla.fenix.HomeActivity} has no registered input channel
05-03 23:37:00.751 E/InputDispatcher(760): Window handle Window{466c7c9 u0 org.mozilla.fennec_fdroid/org.mozilla.fenix.HomeActivity} has no registered input channel
05-03 23:37:00.768 I/Gecko   (22544): Exiting due to channel error.
05-03 23:37:00.770 I/Zygote  (434): Process 22469 exited due to signal 9 (Killed)
05-03 23:37:00.793 I/libprocessgroup(760): Successfully killed process cgroup uid 10081 pid 22469 in 88ms
05-03 23:37:00.793 W/ActivityTaskManager(760): Force removing ActivityRecord{3c8421 u0 org.mozilla.fennec_fdroid/org.mozilla.fenix.HomeActivity t451}: app died, no saved state
05-03 23:37:00.802 E/InputDispatcher(760): Window handle Window{466c7c9 u0 org.mozilla.fennec_fdroid/org.mozilla.fenix.HomeActivity} has no registered input channel
05-03 23:37:00.822 E/InputDispatcher(760): Window handle Window{466c7c9 u0 org.mozilla.fennec_fdroid/org.mozilla.fenix.HomeActivity} has no registered input channel
05-03 23:37:00.835 E/InputDispatcher(760): Window handle Window{466c7c9 u0 org.mozilla.fennec_fdroid/org.mozilla.fenix.HomeActivity} has no registered input channel
```

### Logs 2021-03-20 / crash at 2:25

```
03-20 02:22:00.554 I/a.fennec_fdroi(28763): Background young concurrent copying GC freed 153048(6260KB) AllocSpace objects, 3(60KB) LOS objects, 32% free, 9086KB/13MB, paused 224us total 297.854ms
03-20 02:22:00.946 I/a.fennec_fdroi(28763): Background concurrent copying GC freed 122910(5594KB) AllocSpace objects, 23(664KB) LOS objects, 49% free, 7672KB/14MB, paused 115us total 126.906ms
03-20 02:22:02.031 I/a.fennec_fdroi(28763): Background young concurrent copying GC freed 65470(3570KB) AllocSpace objects, 24(3060KB) LOS objects, 32% free, 10MB/14MB, paused 730us total 119.567ms
03-20 02:22:04.612 I/a.fennec_fdroi(28763): Waiting for a blocking GC ProfileSaver
03-20 02:22:04.700 I/a.fennec_fdroi(28763): Background concurrent copying GC freed 54491(3634KB) AllocSpace objects, 32(6232KB) LOS objects, 49% free, 9911KB/19MB, paused 115us total 290.630ms
03-20 02:22:04.700 I/a.fennec_fdroi(28763): WaitForGcToComplete blocked ProfileSaver on HeapTrim for 87.940ms
03-20 02:22:08.383 I/a.fennec_fdroi(28763): Background young concurrent copying GC freed 48312(3072KB) AllocSpace objects, 49(9740KB) LOS objects, 61% free, 7711KB/19MB, paused 5.209ms total 103.188ms
03-20 02:22:19.344 I/a.fennec_fdroi(28763): Background concurrent copying GC freed 158249(8038KB) AllocSpace objects, 46(4836KB) LOS objects, 49% free, 7885KB/15MB, paused 109us total 182.369ms
03-20 02:22:31.771 V/chatty  (28763): uid=10081(org.mozilla.fennec_fdroid) HeapTaskDaemon identical 10 lines
03-20 02:22:46.322 I/a.fennec_fdroi(28763): Background concurrent copying GC freed 207650(8693KB) AllocSpace objects, 6(964KB) LOS objects, 49% free, 6783KB/13MB, paused 182us total 167.197ms
03-20 02:23:08.308 E/InputDispatcher(765): Window handle Window{7a2572c u0 org.mozilla.fennec_fdroid/org.mozilla.fenix.HomeActivity} has no registered input channel
03-20 02:23:31.108 I/a.fennec_fdroi(28763): Background concurrent copying GC freed 189756(7409KB) AllocSpace objects, 4(116KB) LOS objects, 49% free, 6921KB/13MB, paused 183us total 425.727ms
03-20 02:23:31.973 I/WindowManager(765): WIN DEATH: Window{1430673 u0 org.mozilla.fennec_fdroid/org.mozilla.fenix.HomeActivity}
03-20 02:23:31.979 W/InputDispatcher(765): channel '1430673 org.mozilla.fennec_fdroid/org.mozilla.fenix.HomeActivity (server)' ~ Consumer closed input channel or an error occurred.  events=0xd
03-20 02:23:31.979 E/InputDispatcher(765): channel '1430673 org.mozilla.fennec_fdroid/org.mozilla.fenix.HomeActivity (server)' ~ Channel is unrecoverably broken and will be disposed!
03-20 02:23:31.979 I/ActivityManager(765): Process org.mozilla.fennec_fdroid (pid 28763) has died: fore TOP 
03-20 02:23:31.979 E/InputDispatcher(765): Window handle Window{1430673 u0 org.mozilla.fennec_fdroid/org.mozilla.fenix.HomeActivity} has no registered input channel
03-20 02:23:32.052 W/InputDispatcher(765): Attempted to unregister already unregistered input channel '1430673 org.mozilla.fennec_fdroid/org.mozilla.fenix.HomeActivity (server)'
03-20 02:23:32.092 E/InputDispatcher(765): Window handle Window{1430673 u0 org.mozilla.fennec_fdroid/org.mozilla.fenix.HomeActivity} has no registered input channel
03-20 02:23:32.142 E/InputDispatcher(765): Window handle Window{1430673 u0 org.mozilla.fennec_fdroid/org.mozilla.fenix.HomeActivity} has no registered input channel
03-20 02:23:32.158 E/InputDispatcher(765): Window handle Window{1430673 u0 org.mozilla.fennec_fdroid/org.mozilla.fenix.HomeActivity} has no registered input channel
03-20 02:23:32.173 W/ActivityTaskManager(765): Force removing ActivityRecord{e9ea932 u0 org.mozilla.fennec_fdroid/org.mozilla.fenix.HomeActivity t2243}: app died, no saved state
03-20 02:23:32.175 E/InputDispatcher(765): Window handle Window{1430673 u0 org.mozilla.fennec_fdroid/org.mozilla.fenix.HomeActivity} has no registered input channel
03-20 02:23:32.192 E/InputDispatcher(765): Window handle Window{1430673 u0 org.mozilla.fennec_fdroid/org.mozilla.fenix.HomeActivity} has no registered input channel
03-20 02:23:32.208 E/InputDispatcher(765): Window handle Window{1430673 u0 org.mozilla.fennec_fdroid/org.mozilla.fenix.HomeActivity} has no registered input channel
03-20 02:23:32.225 E/InputDispatcher(765): Window handle Window{1430673 u0 org.mozilla.fennec_fdroid/org.mozilla.fenix.HomeActivity} has no registered input channel
03-20 02:23:32.242 E/InputDispatcher(765): Window handle Window{1430673 u0 org.mozilla.fennec_fdroid/org.mozilla.fenix.HomeActivity} has no registered input channel
03-20 02:23:32.265 E/InputDispatcher(765): Window handle Window{1430673 u0 org.mozilla.fennec_fdroid/org.mozilla.fenix.HomeActivity} has no registered input channel
03-20 02:23:32.275 E/InputDispatcher(765): Window handle Window{1430673 u0 org.mozilla.fennec_fdroid/org.mozilla.fenix.HomeActivity} has no registered input channel
03-20 02:23:32.292 E/InputDispatcher(765): Window handle Window{1430673 u0 org.mozilla.fennec_fdroid/org.mozilla.fenix.HomeActivity} has no registered input channel
03-20 02:23:32.371 I/ActivityManager(765): Process org.mozilla.fennec_fdroid:tab0 (pid 22581) has died: fore SVC 
03-20 02:23:35.194 I/ActivityManager(765):   fore   TOP        0: org.mozilla.fennec_fdroid (pid 28763) top-activity
03-20 02:23:35.194 I/ActivityManager(765):   fore   BTOP       0: org.mozilla.fennec_fdroid:tab0 (pid 22581) service
03-20 02:23:35.194 I/ActivityManager(765):                        org.mozilla.fennec_fdroid/org.mozilla.gecko.process.GeckoChildProcessServices$tab0<=Proc{28763:org.mozilla.fennec_fdroid/u0a81}
03-20 02:24:46.025 I/ActivityTaskManager(765): START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 pkg=org.mozilla.fennec_fdroid cmp=org.mozilla.fennec_fdroid/.App bnds=[827,1662][1018,1914]} from uid 10137
03-20 02:24:46.079 I/ActivityManager(765): Start proc 28098:org.mozilla.fennec_fdroid/u0a81 for pre-top-activity {org.mozilla.fennec_fdroid/org.mozilla.fennec_fdroid.App}
03-20 02:24:46.507 D/libglean_ffi(28098): glean_core::database: Database path: "/data/user/0/org.mozilla.fennec_fdroid/glean_data/db"
03-20 02:24:46.531 V/chatty  (28098): uid=10081(org.mozilla.fennec_fdroid) identical 2 lines
03-20 02:24:46.608 E/Layer   (464): [Surface(name=AppWindowToken{7c37f79 token=Token{f0e0440 ActivityRecord{d2613c3 u0 org.mozilla.fennec_fdroid/.App t2245}}})/@0xb19daed - animation-leash#0] No local sync point found
03-20 02:24:46.609 E/Layer   (464): [Surface(name=AppWindowToken{7c37f79 token=Token{f0e0440 ActivityRecord{d2613c3 u0 org.mozilla.fennec_fdroid/.App t2245}}})/@0xb19daed - animation-leash#0] No local sync point found
03-20 02:24:46.627 I/ActivityTaskManager(765): START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 pkg=org.mozilla.fennec_fdroid cmp=org.mozilla.fennec_fdroid/org.mozilla.fenix.HomeActivity bnds=[827,1662][1018,1914] (has extras)} from uid 10081
03-20 02:24:46.768 I/NavController(28098): Ignoring popBackStack to destination org.mozilla.fennec_fdroid:id/browserFragment as it was not found on the current back stack
03-20 02:24:46.771 V/chatty  (28098): uid=10081(org.mozilla.fennec_fdroid) identical 1 line
03-20 02:24:47.000 W/a.fennec_fdroi(28098): Accessing hidden field Ljava/lang/Boolean;->value:Z (greylist-max-p, JNI, denied)
03-20 02:24:47.001 W/a.fennec_fdroi(28098): Accessing hidden field Ljava/lang/Integer;->value:I (greylist-max-p, JNI, denied)
03-20 02:24:47.001 W/a.fennec_fdroi(28098): Accessing hidden field Ljava/lang/Double;->value:D (greylist-max-p, JNI, denied)
03-20 02:24:47.047 I/ActivityManager(765): Start proc 28180:org.mozilla.fennec_fdroid:tab0/u0a81 for service {org.mozilla.fennec_fdroid/org.mozilla.gecko.process.GeckoChildProcessServices$tab0}
03-20 02:24:47.049 I/ActivityTaskManager(765): Displayed org.mozilla.fennec_fdroid/org.mozilla.fenix.HomeActivity: +1s2ms
03-20 02:24:47.098 W/a.fennec_fdroi(28098): Accessing hidden field Landroid/view/ScaleGestureDetector;->mSpanSlop:I (greylist-max-p, reflection, denied)
03-20 02:24:47.280 W/a.fennec_fdroi(28098): Accessing hidden field Landroid/widget/EdgeEffect;->mPaint:Landroid/graphics/Paint; (greylist-max-p, reflection, denied)
03-20 02:24:47.562 W/ActivityManager(765): Receiver with filter android.content.IntentFilter@cda17fb already registered for pid 28098, callerPackage is org.mozilla.fennec_fdroid
03-20 02:24:48.238 I/a.fennec_fdroi(28098): Background young concurrent copying GC freed 73914(9401KB) AllocSpace objects, 93(2184KB) LOS objects, 89% free, 2762KB/26MB, paused 4.884ms total 143.320ms
03-20 02:24:48.245 V/chatty  (28098): uid=10081(org.mozilla.fennec_fdroid) identical 1 line
03-20 02:24:48.935 V/chatty  (28098): uid=10081(org.mozilla.fennec_fdroid) identical 20 lines
03-20 02:24:54.941 I/NavController(28098): Ignoring popBackStack to destination org.mozilla.fennec_fdroid:id/tabTrayDialogFragment as it was not found on the current back stack
03-20 02:24:55.643 W/a.fennec_fdroi(28098): Core platform API violation: Ljava/nio/Buffer;->position:I from Landroid/graphics/Bitmap; using JNI
03-20 02:24:55.643 W/a.fennec_fdroi(28098): Core platform API violation: Ljava/nio/Buffer;->_elementSizeShift:I from Landroid/graphics/Bitmap; using JNI
03-20 02:24:58.599 I/a.fennec_fdroi(28098): NativeAlloc concurrent copying GC freed 285560(13MB) AllocSpace objects, 85(6264KB) LOS objects, 56% free, 4660KB/10MB, paused 230us total 167.607ms
03-20 02:24:59.614 E/InputDispatcher(765): Window handle Window{a30295c u0 org.mozilla.fennec_fdroid/org.mozilla.fenix.HomeActivity} has no registered input channel
03-20 02:25:07.236 W/InputDispatcher(765): channel 'fe15482 org.mozilla.fennec_fdroid/org.mozilla.fenix.HomeActivity (server)' ~ Consumer closed input channel or an error occurred.  events=0x9
03-20 02:25:07.237 E/InputDispatcher(765): channel 'fe15482 org.mozilla.fennec_fdroid/org.mozilla.fenix.HomeActivity (server)' ~ Channel is unrecoverably broken and will be disposed!
03-20 02:25:07.237 I/ActivityManager(765): Process org.mozilla.fennec_fdroid (pid 28098) has died: fore TOP 
03-20 02:25:07.240 I/WindowManager(765): WIN DEATH: Window{fe15482 u0 org.mozilla.fennec_fdroid/org.mozilla.fenix.HomeActivity}
03-20 02:25:07.240 W/InputDispatcher(765): Attempted to unregister already unregistered input channel 'fe15482 org.mozilla.fennec_fdroid/org.mozilla.fenix.HomeActivity (server)'
03-20 02:25:07.242 E/InputDispatcher(765): Window handle Window{fe15482 u0 org.mozilla.fennec_fdroid/org.mozilla.fenix.HomeActivity} has no registered input channel
03-20 02:25:07.259 E/InputDispatcher(765): Window handle Window{fe15482 u0 org.mozilla.fennec_fdroid/org.mozilla.fenix.HomeActivity} has no registered input channel
03-20 02:25:07.342 E/InputDispatcher(765): Window handle Window{fe15482 u0 org.mozilla.fennec_fdroid/org.mozilla.fenix.HomeActivity} has no registered input channel
03-20 02:25:07.359 E/InputDispatcher(765): Window handle Window{fe15482 u0 org.mozilla.fennec_fdroid/org.mozilla.fenix.HomeActivity} has no registered input channel
03-20 02:25:07.362 W/ActivityTaskManager(765): Force removing ActivityRecord{f5c4734 u0 org.mozilla.fennec_fdroid/org.mozilla.fenix.HomeActivity t2245}: app died, no saved state
03-20 02:25:07.633 I/ActivityManager(765): Process org.mozilla.fennec_fdroid:tab0 (pid 28180) has died: fore SVC 
```

**Comment by kbrosnan** [comment]
I am not able to reproduce using a Galaxy S5 SM-G900M running Android 6 and a Pixel 2 XL running Android 11. I used both Firefox 88 and Nightly 90 to test.

**Comment by Massedil** [comment]
I tried with a fresh install of Firefox 88 from Play Store and I can easily reproduce : if you scroll up down, left & right very quickly, the screen becomes white and everything is slow. Then, the Firefox app crashes.

I tried with Nightly (90.0a1). I can't reproduce the problem : the website is fluid and I can't get Firefox to crash.

**Comment by Massedil** [comment]
After the video, Firefox crashes and I comeback to my home app.

https://user-images.githubusercontent.com/20357783/117174107-abf82780-adcd-11eb-9752-8a501a187529.mp4

**Comment by kbrosnan** [comment]
Which specific model of S5 do you have? There is the international one that I used and the North American version. They have different CPUs and GPUs.

**Comment by Massedil** [comment]
I have a Galaxy S5 SM-G900F updated to LineageOS 18.1 (Android 11).

**Comment by Massedil** [comment]
Solved with version stable 90.1.1. 

So I think there is a change between 89 and 90.
